src/main/java/org/openpnp/machine/reference/ReferenceNozzleTip.java

```diff
@@ -784,34 +965,49 @@ public Location getCalibratedOffset(double angle) {
                 return new Location(LengthUnit.Millimeters, 0, 0, 0, 0);
             }
 
-            // Make sure the angle is between -180 and 180 - angles can get larger/smaller as +-180 if limitation to 180 degrees is disabled
```

Comment:
This angle norming is now done inside the table calibration system where it is really needed. The model based ones don't need that.